.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -5,6 +5,9 @@ Changes proposed in this pull request:
 - a
 - b
 
+##Testing(briefly mention the tests you wrote in regard to the proposed changes)
+- Time Period of logs used for testing
```

Comment:
@bhuvangupta008 We don't necessarily pass on logs to all the functions.
How about specifying the test input characteristics? If the input is logs, then the characteristic would be: the time period, long length in lines and the file size.
If the input is a graph, then the characteristics would be: number of nodes, number of connections, file size.
If the input is a data structure, then the characteristics would be: number of elements, file size.

Similarly, we can give the characteristics of the output as well.

Feel free to add more items to the description.